cdk/domino_cdk/config/template.py

```diff
@@ -127,7 +128,12 @@ def add_nodegroups(name, count, min_size, instance_types, labels, disk_size=disk
             'registry': S3.Bucket(
                 auto_delete_objects=destroy_on_destroy, removal_policy_destroy=destroy_on_destroy, sse_kms_key_id=None
             ),
-        }
+        },
+        monitoring_bucket=S3.Bucket(
```

Comment:
I think you have a config option for this one. But it is not used here.

Reply:
Explain a bit more, not sure I understand?

Reply:
I probably do not understand how we deal with the optional parameters here. 

monitoring_bucket is optional but it is created here without any comment

Reply:
Ah yeah that's totally fair. I'll add actual documentation.